README.md

```diff
@@ -31,3 +35,29 @@ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+## Docs versioning
+
+The versioning is done in two steps:
+
+1. Delete the entry `Stable` from the file `/versions.json`
```

Comment:
do we still need this step if we call the release with the capitalized 'S' ?

Reply:
yes, this is the only step needed. The capitalized `S` was in `docusaurus.config.js` file. So here it's already the new "way"